common/lib/xmodule/xmodule/js/src/video/07_video_volume_control.js

```diff
@@ -37,35 +37,41 @@ function() {
         /** Step to increase/decrease volume level via keyboard. */
         step: 20,
 
-        template: [
+        videoVolumeControlHtml: HtmlUtils.interpolateHtml(
+            HtmlUtils.HTML([
             '<div class="volume" role="application">',
                 '<p class="sr instructions" id="volume-instructions">',
-                    gettext('Click on this button to mute or unmute this video or press UP or DOWN buttons to increase or decrease volume level.'), // jshint ignore: line
+                    '{volumeInstructions}',
                 '</p>',
                 '<button class="control" aria-disabled="false" aria-describedby="volume-instructions"',
                     '" aria-expanded="false" title="',
-                        gettext('Adjust video volume'),
+                        '{adjustVideoVolume}',
                     '">',
-                    '<span class="icon-fallback-img">',
-                        '<span class="icon fa fa-volume-up" aria-hidden="true"></span>',
-                        '<span class="sr control-text">',
-                            gettext('Volume'),
-                        '</span>',
+                    '<span class="icon fa fa-volume-up" aria-hidden="true"></span>',
+                    '<span class="sr control-text">',
```

Comment:
Since this button has a `title` we can simplify this and remove the `sr` text within it.